nservicebus/sql-persistence/index_usageoracle_sqlpersistence_[2.1,).partial.md

```diff
@@ -0,0 +1,8 @@
+
+### Oracle
+
+Using the [Oracle.ManagedDataAccess NuGet Package](https://www.nuget.org/packages/Oracle.ManagedDataAccess).
+
+snippet: SqlPersistenceUsageOracle
+
+NOTE: The `Enlist=false` setting is required for the [Oracle connection string](https://docs.oracle.com/database/121/ODPNT/featConnecting.htm) to prevent auto-enlistment in a [Distributed Transaction](https://msdn.microsoft.com/en-us/library/windows/desktop/ms681205.aspx) which prevents the persistence from enlisting at the correct moment.
```

Comment:
> which prevents the persistence from enlisting at the correct moment.

Not sure if I follow, so after setting it to False the DTC works correctly or is it not supported by Oracle? not clear from this note.